go.mod

```diff
@@ -1,64 +1,65 @@
 module github.com/snowplow/snowbridge
 
-go 1.22
+go 1.22.0
```

Comment:
Let's bump to 1.23, and update deps with 

```
go get -u
go mod tidy
```

I doubt there will be anything breaking, and we have good test coverage. So just passing all the existing tests will be verification enough. If anything does break, and it's not an easy fix, let's make a pragmatic decisions - assuming there's no vulnerabilities we can push anything time consuming to 3.1.0, it shouldn't be long before that release.